packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -371,12 +376,36 @@ export class AngularCompilerPlugin {
       timeEnd('AngularCompilerPlugin._createOrUpdateProgram.ts.createProgram');
     } else {
       time('AngularCompilerPlugin._createOrUpdateProgram.ng.createProgram');
+
+      const oldProgram = (this._program as Program);
+      if (oldProgram && !this._compilerOptions.enableIvy) {
+        // This is a workaround as NGC doesn't offer a way to
+        // purge summary files for library dependencies.
+        // Note: NGTSC program doesn't implement 'getLibrarySummaries' method.
+        const originalGetLibrarySummaries = oldProgram.getLibrarySummaries;
+
+        oldProgram.getLibrarySummaries = () => {
```

Comment:
I feel a bit concerned with this workaround. It looks like the sort of thing that should be addressed in the VE compiler. @kara where do you think this should be addressed?